server/cmd/run.go

```diff
@@ -196,5 +258,10 @@ func run(ctx context.Context, c *config.Config, lv int) error {
 		errCh <- adminSrv.Run(c.AdminPort)
 	}()
 
+	go func() {
+		log.Info("Starting manager")
+		errCh <- mgr.Start(signals.SetupSignalHandler())
```

Comment:
I might be wrong, but only the manager will be shut down first, and the server won't stop until the manager has finished. However, wouldn’t skipping the graceful shutdown lead to fewer request drops?